addon/utils/is-descriptor.js

```diff
@@ -1,7 +1,6 @@
 export default function isDescriptor(item) {
   return item &&
     typeof item === 'object' &&
-    'writable' in item &&
```

Comment:
This one was weird, could be a bug with Babel, but in a consuming Babel 6 app, when I ran a decorator on a class property with no parameters, the descriptor didn't have the writable property at all:

```
class FooComponent extends Component {
  @service bar
}
```